_includes/doc-side-nav.html

```diff
@@ -31,7 +31,8 @@
 				{% if d.order == nil %}
 				{% assign order = 9999 %}
 				{% endif %}
-        		], url: "{{d.url}}", title: "{{d.title}}", order: {{order}}, overview: "{{d.overview}}"});
+                {% assign url = d.url | remove: "/index.html" %}
```

Comment:
not sure how this works but here also make sure the trailing / is present in the source